tests/SecurityTest.php

```
@@ -64,4 +64,111 @@ public function testBotDetection(): void
+
+        // Verify null token validation returns false
+        $this->assertFalse(SecurityUtils::validateCsrfToken(null));
+    }
+
+    /**
+     * Test dynamic security headers
+     */
+    public function testSecurityHeaders(): void
+    {
+        // Headers are set only if not already sent (which PHPUnit environment allows mock-testing)
+        SecurityUtils::sendSecurityHeaders();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\CmsForNerd\SecurityUtils' in method 'testSecurityHeaders'. (undefined)

(StaticAccess)